modules/htlc/exported/exported.go

```diff
@@ -0,0 +1,23 @@
+package exported
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+var (
+	NewParamSetPair = paramtypes.NewParamSetPair
+)
+
+type (
+	ParamSet      = paramtypes.ParamSet
+	ParamSetPairs = paramtypes.ParamSetPairs
```

Comment:
Remove `NewParamSetPair` and `paramSetParis` as they are never used. Make same modifications to `irismod/coinswap` and  `irismod/farm`.